src/main/scala/lms/transformation/distributed_tensor/distributedTensorConv.scala

```diff
@@ -111,6 +129,16 @@ trait FixedSizeDistributedTensorConvTypeLess extends FixedSizeDistributedTensorM
           val b_grad = ConvBackwardFilter(x, y, gradMap(s), params, anno, pos)
           Accumulate(gradMap(b), b_grad, anno); ()
         }) +=: backwardNodes
+      
+      case Node(s, "tensor_softmax", tt::Backend.Const(anno:Anno)::(a:Backend.Sym)::Backend.Const(params:SoftmaxParam)::_, _) =>
+        implicit val pos = Adapter.oldSourceMap(s)
+        forwardNodes += node
+        (() => {
+            val x = new TENSOR(transform(a))
+            val grad = SoftmaxBackward(x, gradMap(s), params, anno, pos)
+            Accumulate(gradMap(a), grad, anno); ()
+           ()
```

Comment:
I am not sure about this one:
should it be
`val grad = SoftmaxBackward(x, gradMap(s), params, anno, pos)`
or
`val grad = SoftmaxBackward(new TENSOR(transform(s)), gradMap(s), params, anno, pos)`?

The API https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnSoftmaxBackward seems to be using `yDesc` and `y`, but I am not certain.

Reply:
I think it's the second one, but I have an error when running on the server.